src/js/components/Popup/PopupAction.tsx

```diff
@@ -69,13 +69,20 @@ export default class PopUpAction extends React.Component<Props, State> {
               {this.props.actions.map(action => {
                 return (
                   <div key={action.text}>
-                    <p
+                    <button
+                      tabIndex={0}
                       className="u-padEndsSm u-padSidesMd"
-                      style={{ cursor: "pointer" }}
+                      style={{
+                        cursor: "pointer",
+                        appearance: "none",
+                        border: "none",
```

Comment:
@bradcypert looks like you'll need to set background to none too.

<img width="214" alt="Screen Shot 2019-09-10 at 10 50 18 AM" src="https://user-images.githubusercontent.com/1189536/64633766-12d6fa80-d3b9-11e9-9838-8f22eda35f44.png">

Reply:
Updtaed. What browser were you testing in?